etc/profile-a-l/alpine.profile

```diff
@@ -7,6 +7,9 @@ include alpine.local
 # Persistent global definitions
 include globals.local
 
+# Workaround for bug https://github.com/netblue30/firejail/issues/2747
+# firejail --ignore="shell none" --private-bin=bash --noblacklist=/bin/bash --profile=alpine bash -c '(alpine)'
```

Comment:
Is `--ignore="shell none"` necessary?

Reply:
you need bash to launch alpine in a subshell

Reply:
> you need bash to launch alpine in a subshell

Yes you need a shell (bash or sh IDC) in the sandbox that launches alpine (IDK why but if this is the workaround ...). But why do you need to ignore `shell none` if you `bash -c`?

Reply:
`shell none` does not disable/blacklist (all) shells inside the sandbox (for that we have `disable-shell.inc`). It changes the way firejail starts the sandbox.
`firejail --noprofile --shell=none /bin/bash -c firefox` is mostly the same as `firejail --noprofile firefox` (note that there are edge cases and the /bin/bash vs. $SHELL bug).

Reply:
If i remove it, i get this (/usr/local/bin/bash doesn't exist)

```
Error: execute permission denied for /usr/local/bin/bash
Error: no suitable bash executable found
```

Reply:
Why is your bash at `/usr/local/bin/bash`? What does `firejail --ignore=private-bin --profile=alpine which -a bash` and  `firejail --ignore=private-bin --profile=alpine ls -l ADD_BASH_PATH_HERE` show?

Reply:
there's no bash executable there

```
$ firejail --ignore=private-bin --profile=alpine which -a bash
Cannot start application: Permission denied
```

```
$ firejail --ignore=private-bin --profile=alpine ls -l /bin/bash
-r-------- 1 nobody nogroup 0 Jun 10 12:14 /bin/bash
```


```
$ firejail --ignore=private-bin --profile=alpine ls -l /usr/local/bin/bash
ls: cannot access '/usr/local/bin/bash': No such file or directory
```







firejail from the ppa, 32 bit ubuntu 18.04
```
$ firejail --version
firejail version 0.9.64.4

Compile time support:
	- AppArmor support is enabled
	- AppImage support is enabled
	- chroot support is enabled
	- D-BUS proxy support is enabled
	- file and directory whitelisting support is enabled
	- file transfer support is enabled
	- firetunnel support is enabled
	- networking support is enabled
	- overlayfs support is disabled
	- private-home support is enabled
	- private-cache and tmpfs as user enabled
	- SELinux support is enabled
	- user namespace support is enabled
	- X11 sandboxing support is enabled
```

Reply:
This isn't caused by `shell=none` (try `firejail --noprofile --shell=none ls -l /bin/bash`). `ingore=shell=none` makes it work as a (not protable?) side effect I guess.

```suggestion
# firejail --private-bin=sh --include='${CFG}/allow-bin-sh.inc' --profile=alpine sh -c '(alpine)'
```